### DISCUSSION TITLE
Add Field descriptions

### DISCUSSION THREAD

**Discussion by MurzNN** [keystonejs/keystone#5875] [Feature requests]
Very often in Admin UI we need to describe, how to fill the exact field rightly (via short or medium text description below the field). At now we can set only custom field label via `label` option, but this is the field title, and placing long text to it is not so good idea. 

To solve this problem will be good to add the `description` option to field, that allows to set custom text (html formatted) to show as description in Admin UI list item edit form below the it's input box.

**Comment by bladey** [comment]
Excellent idea, thanks @MurzNN.

**Reply by VinayaSathyanarayana** [reply]
Similar support was available in earlier versions of Keystone

**Comment by JMLX42** [comment]
There is a `description` field already for schemas: https://keystonejs.com/docs/apis/schema#description

> The description option defines a string which will be used as a description in the Admin UI and GraphQL API docs. This option can be individually overridden by the graphql.description or ui.description options.

But unlike what the doc says, setting this field has no impact on the Admin UI.

**Reply by dcousens** [reply]
That is for `lists`,  but you are right that this is a bug, it isn't actually shown in the frontend

**Reply by JMLX42** [reply]
Indeed. I just opened a separate issue for the schema description bug: https://github.com/keystonejs/keystone/issues/7325

**Comment by gwyneplaine** [comment]
As per @dcousens  and @JMLX42 this is a bug, we're looking to implement the documented behavior shortly.
